src/uct/ib/base/ib_iface.h

```diff
@@ -357,4 +359,59 @@ void uct_ib_iface_set_max_iov(uct_ib_iface_t *iface, size_t max_iov)
 }
 
 
+static UCS_F_ALWAYS_INLINE
+void uct_ib_iface_fill_ah_attr_from_addr(
```

Comment:
can we keep just one function? (these two are very similar now)

Reply:
unfortunately no: there are different incoming arguments and processing logic. I agree that there is portion of similar code, but split it to few functions will reduce "keep-it-simple" level

Reply:
Then can you make uct_ib_iface_fill_ah_attr_from_addr to be a wrapper of uct_ib_iface_fill_ah_attr_from_gid_lid?